internal/vpn/client.go

```diff
@@ -214,14 +214,14 @@ func (c *Client) Serve() error {
 		defer close(connToTunDoneCh)
 
 		if _, err := io.Copy(tun, c.conn); err != nil {
-			c.log.WithError(err).Errorf("Error resending traffic from TUN %s to VPN server", tun.Name())
+		fmt.Printf("Error resending traffic from TUN %s to VPN server. error = %v", tun.Name(), err))
```

Comment:
This needs to be formatted. Could you please run `goimports` for this file? There's also an extra bracket at the end of the line.